src/com/facebook/buck/jvm/groovy/GroovycStep.java

```diff
@@ -61,7 +70,6 @@
 
   @Override
   public int execute(ExecutionContext context) throws IOException, InterruptedException {
-
```

Comment:
If you can figure out how to make that a project setting, we'd take a PR for that too :)